### ISSUE TITLE
[Event]: release

### ISSUE THREAD

**Issue by aloispichler** [hyprland-community/hyprland-rs#267]
### What is the name of the event

we need a new release: https://github.com/H3rmt/hyprswitch/issues/47

**Comment by diniamo** [comment]
Yes, different projects need it too, but yavko doesn't really care.

**Comment by yavko** [comment]
> Yes, different projects need it too, but yavko doesn't really care.

I do care, I don't have time

**Comment by yavko** [comment]
Hi all, I've found the time this weekend because of labor day, here is the pr with all the new features and implementations #177, if you want a release to come out faster, some testing would be very helpful!
